app/views/events/_past.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="m-5">
+<% if @past_events.empty? %>
+  <%= "There're no previous events" %>
+<% else %>
+<% @past_events.each do |inv| %>
+  <div class="card" style="width: 18rem;">
+    <div class="card-body">
+      <h5 class="card-title"><%= "Event in #{inv.location } " %> </h5>
+      <p class="card-text"> <%= "at #{inv.date.to_formatted_s(:rfc822)}." %></p>
+      <%= link_to 'View event', event_show_path(inv)  %>
+  </div>
+</div>
+<% end %>
+<% end %>
```

Comment:
- Move the if/else statement to a helper method that you call here and render the markup within the helper method using a partial